test/test_eth_registrar_controller.js

```diff
@@ -58,150 +61,192 @@ contract('ETHRegistrarController', function (accounts) {
 		await ens.setSubnodeOwner('0x0', sha3('eth'), baseRegistrar.address);
 
 		priceOracle = await SimplePriceOracle.new(1);
+		referrers = await ACL.new();
 		controller = await ETHRegistrarController.new(
 			baseRegistrar.address,
 			priceOracle.address,
 			600,
 			86400,
+			referrers.address,
 			{from: ownerAccount});
 			await baseRegistrar.addController(controller.address, {from: ownerAccount});
-		});
-
-		const checkLabels = {
```

Comment:
Did you change indentation or something? This makes comparing the diff super difficult

Reply:
The indentation previously was incorrect - this section was indented too far. The GitHub diff should highlight any changes in the red area separately.